src/index.js

```diff
@@ -8,13 +8,22 @@ apply(listTask, container);
 
 const addTodoBtn = document.querySelector('#add');
 addTodoBtn.addEventListener('click', () => {
+  const id = `id${Math.random().toString(16).slice(2)}`;
   const description = document.querySelector('.todo').value.trim();
   const completed = false;
   const index = listTask.list.length + 1;
-  const newTodo = { description, completed, index };
+  const newTodo = {
+    id, description, completed, index,
+  };
   if (description) {
     listTask.addTask(newTodo);
     apply(listTask, container);
   }
   document.querySelector('.todo').value = '';
 });
```

Comment:
- [ ] **Optional:** To have a better user experience you could implement adding items with the Enter key. Consider the code block below:

```
document.addEventListener("keydown", (key) => {
  console.log(key.key);
  if (key.key === "Enter") {
    // add item to list
    console.log("Im adding");
  }
});

```

And this [resource.](https://www.section.io/engineering-education/keyboard-events-in-javascript/)